core/server/common/src/main/webapp/browse-body.jsp

```diff
@@ -17,35 +17,17 @@
 
 <jsp:include page="header-scripts.jsp" />
 
-<script>
-  var currentDir = "<%= request.getAttribute("currentPath").toString() %>";
-  function getInputPath() {
-    var path = $.trim($("#pathInput").val());
-    return path;
-  }
-
-  function changeDir() {
-    var path = getInputPath();
-    path = encodeURI(path);
-    window.location.href = "./browse?path=" + path;
-  }
+<link rel="stylesheet" href="css/ui-grid.css" type="text/css">
```

Comment:
- I've done it, PTAL. Thanks for your link, I learn from there.
- I use maven plugin `exec-maven-plugin` to execute `npm install angular-ui-grid` in ${project.basedir}/src/main/webapp/build
- Because of files will generate to  the build folder, so I add the path to the license check exclude paths.
- I've test this change use `mvn  install -Dhadoop.version=2.7.1 -Pyarn,spark -DskipTests -Dfindbugs.skip -Dmaven.javadoc.skip -Dcheckstyle.skip`, it worked well.